drainer/checkpoint/util.go

```diff
@@ -68,7 +68,11 @@ func setDefaultConfig(cfg *Config) {
 		cfg.Db.User = "root"
 	}
 	if cfg.Schema == "" {
-		cfg.Schema = "tidb_binlog"
+		if cfg.CheckpointType == "oracle" {
+			cfg.Schema = cfg.Db.User
```

Comment:
emmmmm. Can you explain why for this?

Reply:
user in oracle like the schema in TiDB

Reply:
If oracle has special requirement, I prefer to force user to set the checkpoint schema